package protoFile.fight.showAllCards;
message showAllCards{
     required int32 signal =1; 
}